infinex-patron/vesting/1-patron-vesting-configuration.ethereum.toml

```diff
@@ -0,0 +1,45 @@
+# pnpm cannon build infinex-patron/vesting/1-patron-vesting-configuration.ethereum.toml --dry-run --chain-id 1
+
+name = "infinex-patron-vesting-configuration"
+preset = "config"
+version = "1.0.1"
+description = "Configuration for the Infinex Patron Vesting"
+
+[pull.InfinexPatronDistributor]
+source = "infinex-patron-vesting:1.0.1@patron-vesting"
+
+[invoke.SetTier1StartTime]
+target = ["<%= imports.InfinexPatronVesting.imports.InfinexPatronVesting.contracts.PatronVesting.address %>"]
+func = "setTierStartTime"
+args = [1, 0] # 
```

Comment:
why are we doing this? also there's a `#` every where which looks like there was a comment before but got remove

Reply:
I think there was a comment before and then it got removed